apps/playground-api/src/setups/setup.gov.ts

```diff
@@ -64,12 +64,7 @@ export class SetupGov extends PlaygroundSetup<Record<string, any>> {
   }
 
   async has (each: any): Promise<boolean> {
-    const key = Object.keys(each)[0]
-    const gov = await this.client.masternode.getGov(key)
-    if (Object.keys(gov[key]).length > 0) {
-      return true
-    }
-    return false
+    return false // for just overwrite
```

Comment:
If return `false` is something expected, we should remove this function

Reply:
And the function name is `has`,  it is a positive word so return true seems more sensible